### PR TITLE
Update testlist for CESM3. 

### PR DESCRIPTION
**Pull request by fischer-ncar**
### Description of changes
Moved testing from POP to MOM.  

### Specific notes
The fullactive testlist is still evolving.

Contributors other than yourself, if any:

Fixes: [Github issue #s] And brief description of each issue.

User interface changes?: [ No/Yes ]
[ If yes, describe what changed, and steps taken to ensure backward compatibility ]

Testing performed (automated tests and/or manual tests):

### CODE REVIEW

**Comment by klindsay28** [comment]
Should we be running tests with MOM on the grid `t232`, instead of `t061`?
We switched from `t061` to `t232` awhile ago.

**Comment by fischer-ncar** [comment]
I didn't know that.  I can switch to t232.

File: cime_config/testlist_allactive.xml [hunk 1]
```diff
@@ -142,7 +133,7 @@
       <option name="wallclock"> 01:30:00 </option>
     </options>
   </test>
-  <test name="PFS" grid="f09_g17" compset="B1850" testmods="allactive/defaultio">
+  <test name="PFS" grid="f09_t232" compset="B1850" testmods="allactive/defaultio">
```
Comment (jedwards4b) [inline cime_config/testlist_allactive.xml:136]: Do we have and PFS tests on the BLT1850_v0c compset?   That would be preferred I think.

**Review by jedwards4b — APPROVED** [review]
Overall I think we still have too many fv dycore tests and not enough ne30pg3 tests - but I think we can continue to adjust in upcoming tags - maybe adding ne30pg3 in one tag and then removing f09 in the next tag so that there is some overlap in the baselines.

File: cime_config/testlist_allactive.xml (hunk 1 above)
Comment (fischer-ncar) [reply cime_config/testlist_allactive.xml]: Changed it to ne30pg3_t232 BLT1850_v0c

Comment (jedwards4b) [reply cime_config/testlist_allactive.xml]: great thanks

**Pull request merged**